packages/astro/test/units/test-utils.js

```diff
@@ -187,7 +188,7 @@ export function createBasicEnvironment(options = {}) {
 		clientDirectives: getDefaultClientDirectives(),
 		resolve: options.resolve ?? ((s) => Promise.resolve(s)),
 		routeCache: new RouteCache(options.logging, mode),
-		logging: options.logging ?? defaultLogging,
+		logging: options.logging ?? defaultLogger,
```

Comment:
I think the two `logging` here should be `logger` since the `Environment` types had it updated too.